app/frontend/styles/application.scss

```diff
@@ -34,6 +34,7 @@ $govuk-image-url-function: frontend-image-url;
 @import 'components/links';
 @import 'components/location_finder';
 @import 'components/modal_dialogue';
+@import 'components/moj_scrollable_pane';
```

Comment:
isnt this already imported on line 13 of this file?

Reply:
ignore that :)

Reply:
That's the original unedited version